quickstarts/aws/amazon-vpc-nat-gateway/config.yml

```diff
@@ -0,0 +1,37 @@
+slug: amazon-vpc-natgateway
+description: |
+  ## Monitor your NAT Gateway in New Relic's observability platform
+  Our NAT Gateway integration reports data from your NAT Gateway service to your New Relic account using AWS CloudWatch Metric Streams.
+  ## Get started!
+  This integration uses [AWS CloudWatch Metric streams](https://docs.newrelic.com/docs/infrastructure/amazon-integrations/connect/aws-metric-stream/) to get NAT Gateway traffic data into New Relic.
+  ## About this integration
+  With this integration you can monitor key traffic information about your NAT Gateway.
+  - Track your connection established rate
+  - View idle connection timeouts
+  - Track Byte traffic
+  - Monitor your active connections over timeouts
```

Comment:
The bit about "over timeouts" is a little unclear to me

Reply:
should be over time, not timeouts. I think VSCode auto completed that one for me.